lib/widget/dropdown.dart

```diff
@@ -197,6 +207,24 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
+        item.iconname != null
+            ? results.add(
+          DropdownMenuItem(
+            child: Row(
+              children: [
+                iconframework.Icon(item.iconname),
+                const SizedBox(
+                  width: 10.0,
```

Comment:
size should not be hardcoded. It can default to 10 but user should be able to override in yaml. We talked about this. @hamxa152